doc/Language/pod.pod6

```diff
@@ -1,10 +1,10 @@
 =begin pod :kind("Language") :subkind("Language") :category("reference")
 
-=TITLE Perl 6 pod
+=TITLE Raku pod
 
 =SUBTITLE An easy-to-use markup language for documenting Perl modules and programs
```

Comment:
This should probably refer to Raku

Reply:
See this discussion on @lizmats's PR: https://github.com/perl6/problem-solving/commit/07c423e825648322490bcea68e0deba1c2ad6a6b#r34680556

Reply:
I'm going to solve that issue soon anyway: see #1923

Reply:
Updated.